### ISSUE TITLE
No wheels for Linux aarch64 (ARM)

### ISSUE THREAD

**Issue by dan-blanchard** [bodono/scs-python#111]
I have noticed that when I try to install scs-python on a Linux ARM (aarch64) machine, there are no wheels for it on PyPI.

I looked into it briefly, and it appears if you follow the instructions [here](https://cibuildwheel.pypa.io/en/stable/faq/#emulation) you should be able to pretty easily add those to your build matrix.

**Comment by bodono** [comment]
On pypi I see arm64 wheels: https://pypi.org/project/scs/#files

Are they not working? Or are you saying that the problem is they are for mac-os rather than linux?

**Comment by dan-blanchard** [comment]
Yeah, those are just for macOS. Linux ARM ones are called aarch64, and have to be built via emulation like in the docs I linked.

**Comment by matteoettam09** [comment]
Any progress on this? I am also having issues on Linux aarch64 (ARM):

```
uv pip install scs
Resolved 3 packages in 24ms
  × Failed to download and build `scs==3.2.7`
  ╰─▶ Build backend failed to build wheel through `build_wheel` (exit status: 1)

      [stdout]
      The Meson build system
      Version: 1.6.0
      Build type: native build
      Project name: scs
      Project version: undefined
      C compiler for the host machine: cc (gcc 12.2.0 "cc (Raspbian 12.2.0-14+rpi1) 12.2.0")
      C linker for the host machine: cc ld.bfd 2.40
      Host machine cpu family: arm
      Host machine cpu: arm
      Program python found: YES (/home/matteoettam09/.cache/uv/builds-v0/.tmp1SkP1i/bin/python)
      Library openblas found: YES

      import numpy as np
      try:
        incdir = os.path.relpath(np.get_include())
      except Exception:
        incdir = np.get_include()
      print(incdir)
      '` failed with status 1.
```

**Comment by matteoettam09** [comment]
This may do the trick here as well?

https://github.com/cvxpy/cvxpy/blob/096bdf1eb7c05e6c88d24d8435dceae2a2d1f816/.github/workflows/build.yml#L139

**Comment by Transurgeon** [comment]
> This may do the trick here as well?
> 
> https://github.com/cvxpy/cvxpy/blob/096bdf1eb7c05e6c88d24d8435dceae2a2d1f816/.github/workflows/build.yml#L139

Do you mean adding the changes in that PR ([cvxpy#2054](https://github.com/cvxpy/cvxpy/pull/2054/files)) to the scs CI?
If so, I think that's a great idea and should hopefully fix the build issues :).

**Comment by matteoettam09** [comment]
Yes, that's what I was thinking. But didn't spend a lot of time looking at the current build of scs-python (nor cvxpy), just looked into exiting issues. 

May find some time to look into this during the weekend, thanks. Open to alternatives, in the meantime.

**Comment by bodono** [comment]
This should be fixed now, see discussion here: https://github.com/bodono/scs-python/pull/121
